### PR TITLE
use ModelHelper for prediction

### PR DESCRIPTION
**Pull request by skatedplrn**
_Provide a description of what has been changed_

### Checklist

- [ ] Documentation has been added to relevant sections in the `docs/` directory
- [ ] Issue(s) exist for this PR and are referenced below (e.g. "Fixes #32")
- [ ] Tests have been added

Fixes #

### CODE REVIEW

**Comment by knobotz** [comment]
@skatedplrn which story is this related to?

File: a2ml/api/auger/impl/mparts/predict.py [hunk 1]
```diff
@@ -122,6 +115,7 @@ def _docker_run_predict(self, filename, threshold, model_path):
                 'Running model in deeplearninc/'
                 'auger-ml-worker:%s' % docker_tag)
             result_file = subprocess.check_output(command, stderr=subprocess.STDOUT, shell=True)
+            print(result_file)
```
Comment (holyketzer) [inline a2ml/api/auger/impl/mparts/predict.py:118]: log?

File: a2ml/api/base_a2ml.py [hunk 2]
```diff
@@ -1,13 +1,15 @@
-from a2ml.api.utils.crud_runner import CRUDRunner
-from a2ml.api.utils.provider_runner import ProviderRunner
-from a2ml.api.utils.remote_runner import RemoteRunner
-
 
 class BaseA2ML(object):
     def build_runner(self, ctx, provider, object_name=None, force_local=False):
         if ctx.config.get('use_server') == True and not force_local:
+            from a2ml.api.utils.remote_runner import RemoteRunner
```
Comment (holyketzer) [inline a2ml/api/base_a2ml.py:5]: I don't see the point to moving any import lines inside the functions, it makes the codebase more complex, it's hard to judge about file dependancies when you check the top of file it doesn't contains all of them.

Comment (knobotz) [reply a2ml/api/base_a2ml.py:5]: Nested imports should only happen if the import is large or if it results in importing something that isn't installed by default. So in this case, @holyketzer does [remote_runner.py](https://github.com/augerai/a2ml/blob/master/a2ml/api/utils/remote_runner.py) import anything that isn't installed via the [default dependency group](https://github.com/augerai/a2ml/blob/master/setup.py#L30-L46)? If so, then the nested import approach is correct IMHO.

File: a2ml/api/auger/impl/cloud/pipeline.py [hunk 3]
```diff
@@ -37,6 +37,7 @@ def predict(self, records, features, threshold=None):
         prediction_api = AugerPredictionApi(self.ctx, self)
         prediction_properties = \
             prediction_api.create(records, features, threshold)
+        print(prediction_properties)
```
Comment (knobotz) [inline a2ml/api/auger/impl/cloud/pipeline.py:40]: Use log debug/info here.

**Pull request merged**